ricecooker/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 __author__ = 'Learning Equality'
 __email__ = 'jordan@learningequality.org'
-__version__ = '0.3.5'
\ No newline at end of file
+__version__ = '0.3.5c12'
```

Comment:
Is this the latest version? Maybe it's time to remove the beta-ness and upgrade to `0.3.6`!